src/main/java/org/folio/rest/utils/ExpirationTool.java

```diff
@@ -123,11 +122,10 @@ public static void doExpiration(Vertx vertx, Context context) {
     context.runOnContext(v -> {
       try {
         PostgresClient pgClient = PostgresClient.getInstance(vertx, tenant);
-        pgClient.setIdField("id");
         Criteria idCrit = new Criteria();
         idCrit.addField(USER_ID_FIELD);
         idCrit.setOperation("=");
-        idCrit.setValue(user.getId());
+        idCrit.setVal(user.getId());
         Criterion criterion = new Criterion(idCrit);
 
         pgClient.update(TABLE_NAME_USERS, user, criterion, true, updateReply -> {
```

Comment:
Can you drop Criteria/Criterion and directly use the method that takes the id parameter?

    pgClient.update(TABLE_NAME_USERS, user, user.getId(), updateReply -> {